tests/fixtures/metagenome_assembly_set.json

```diff
@@ -14,19 +14,20 @@
     "scaf_logsum": 5934400,
     "scaf_pct_gt50K": 1.7355337,
     "gap_pct": 0.00334,
-    "git_url": "https://github.com/microbiomedata/mg_annotation/releases/tag/0.1",
+    "git_url": "https://github.com/microbiomedata/metaAssembly",
+    "version": "v1.0.3",
     "has_output": [
-      "nmdc:37573bca240f88091720ae61ae5c9452",
-      "nmdc:220c85a9e2d470157aed7a7556f85656",
-      "nmdc:c3298d022868e8b0e7cfb783015b93ed",
-      "nmdc:3fc528f0105fc163876d68df26d35b29",
-      "nmdc:da186671c5a581af0a7d195bd857c871"
+      "nmdc:dobj-11-contigs",
+      "nmdc:dobj-11-cover-stats",
```

Comment:
I dont think the records with three dashes in them are going to be schema-valid. I would like to see integration tests where the example data is validated against the schema, either with the python package or linkml-validate

Reply:
errors from runtime json:validate.
{
  "result": "errors",
  "detail": {
    "metagenome_annotation_activity_set": [
      "'nmdc:dobj-11-proteins-faa' does not match '^(nmdc):dobj-([0-9][a-z]{0,6}[0-9])-([A-Za-z0-9]{1,})$'",
      "'nmdc:dobj-11-scaffold-lineage' does not match '^(nmdc):dobj-([0-9][a-z]{0,6}[0-9])-([A-Za-z0-9]{1,})$'",
      "'nmdc:dobj-11-ko-ec' does not match '^(nmdc):dobj-([0-9][a-z]{0,6}[0-9])-([A-Za-z0-9]{1,})$'",
      "'nmdc:dobj-11-product-names' does not match '^(nmdc):dobj-([0-9][a-z]{0,6}[0-9])-([A-Za-z0-9]{1,})$'",
      "'nmdc:dobj-11-gene-phylo' does not match '^(nmdc):dobj-([0-9][a-z]{0,6}[0-9])-([A-Za-z0-9]{1,})$'",
      "'nmdc:dobj-11-annotation-stats' does not match '^(nmdc):dobj-([0-9][a-z]{0,6}[0-9])-([A-Za-z0-9]{1,})$'",
      "'nmdc:dobj-11-contig-mapping' does not match '^(nmdc):dobj-([0-9][a-z]{0,6}[0-9])-([A-Za-z0-9]{1,})$'",
      "'nmdc:dobj-11-annotation-info' does not match '^(nmdc):dobj-([0-9][a-z]{0,6}[0-9])-([A-Za-z0-9]{1,})$'",
      "'nmdc:omprc-11-test-metag1' does not match '^(nmdc):(omprc)-([0-9][a-z]{0,6}[0-9])-([A-Za-z0-9]{1,})$'"
    ]
  }
}